DependencyInjection/Compiler/TwigExceptionPass.php

```diff
@@ -39,7 +39,11 @@ public function process(ContainerBuilder $container)
         }
 
         if (!$container->has('templating.engine.twig')) {
-            $container->removeDefinition('fos_rest.exception.twig_controller');
+            if ($container->has('twig') && $container->has('fos_rest.exception.twig_controller')) {
+                $container->getDefinition('fos_rest.exception.twig_controller')->replaceArgument(3, $container->getDefinition('twig'));
```

Comment:
If we use `has()` instead of `hasDefinition()` inside the `if` (which is totally fine), we should use `findDefinition()` instead of `hasDefinition()` here.

Reply:
Makes sense; looks like `has()` includes aliases and `getDefinition()` does not, so good to be consistent. All set.